README.md

```diff
@@ -13,11 +13,19 @@
 [![PRs Welcome][prs-badge]][prs]
 
 Atom package to format your JavaScript, Typescript, CSS, Sass, and JSON using [Prettier](https://github.com/prettier/prettier).
-Comes with powerful optional [ESLint](http://eslint.org/) integration.
+Optionally integrates with [ESLint](http://eslint.org/), [EditorConfig](http://editorconfig.org/), and [stylelint](https://stylelint.io/).
+
+## How to use it
+
+After [installation](#installation) you are ready to use prettier in a way that suits your needs
```

Comment:
- needs a comma after installation
- I think we can drop `you are ready to use prettier in a way that suits your needs`
- we should probably use a colon at the end to indicate we are giving a list as the end of the sentence

Reply:
So a colon instead of a comma?  haha, I'm a little confused

```
After installation:
- ...
- ...
- ...
```